Assets/Huawei/Editor/View/LibrariesTab/HMSLibrariesDrawer.cs

```diff
@@ -1,32 +1,38 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace HmsPlugin
+﻿namespace HmsPlugin
 {
     public class HMSLibrariesDrawer : VerticalSequenceDrawer
     {
         private TabBar _tabBar;
         private TabView _tabView;
         private Toggle.Toggle _appSupportToggle;
+        private Toggle.Toggle _hmsCoreInstallToggle;
 
         public const string AppCompatEnabled = "AppCompat";
+        public const string GooglePlayReleaseEnabled = "GooglePlayRelease";
```

Comment:
We should call it HMSCoreInstalledEnabled

Reply:
It makes sense 👍🏽